include/swift/Runtime/EnvironmentVariables.h

```diff
@@ -0,0 +1,39 @@
+//===--- CompatibiltyOverride.h - Debug variables. --------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Debug behavior conditionally enabled using environment variables.
+//
+//===----------------------------------------------------------------------===//
+
+#include "../Basic/Lazy.h"
+
+namespace swift {
+namespace Environment {
```

Comment:
Nit: the `swift::Environment` makes me twitch (`swift::environment` is much more C++-esque).  I think that adding the explicit `runtime` in the namespace is really nice to ensure that we do not cause future ODR violations in mixed language environments.  What do you think of `swift::runtime::environment` for the namespace here?